pebbles/models.py

```diff
@@ -514,118 +511,6 @@ def __init__(self, lock_id):
         self.acquired_at = datetime.datetime.utcnow()
 
 
-class Variable(db.Model):
```

Comment:
Should there be a migration attached that removes the variable table from the db? Am I missing it because it has no corresponding file to compare to?